golang/x/relay/client/cli/query.go

```diff
@@ -324,18 +325,36 @@ func GetCmdGetRequest(queryRoute string, cdc *codec.Codec) *cobra.Command {
 	return &cobra.Command{
 		Use:     "getrequest <id>",
 		Example: "getrequest 12",
-		Long:    "Get a proof request using the associated ID",
+		Long:    "Get a proof request using the associated ID. ID can be an\n\"0x\" prepended hexbyte string or an integer",
 		Args:    cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cliCtx := context.NewCLIContext().WithCodec(cdc)
 
-			id, err := strconv.ParseUint(args[0], 10, 64)
-			if err != nil {
-				return err
+			var idBytes types.RequestID
+			if args[0][:2] == "0x" {
```

Comment:
A large chunk here is getting the ID from a string, by checking if it's hex or not. Can we abstract that into a utility function?